### PR TITLE
Include uid in session.start & upload events

### PR DESCRIPTION
**Pull request by xacrimon**
This PR makes sure we include UUID identifiers in all `session.start` and `session.upload` events.

### CODE REVIEW

File: lib/events/auditlog.go [hunk 1]
```diff
@@ -348,6 +349,7 @@ func (l *AuditLog) UploadSessionRecording(r SessionRecording) error {
 	}
 	l.log.WithFields(log.Fields{"duration": time.Since(start), "session-id": r.SessionID}).Debugf("Session upload completed.")
 	return l.EmitAuditEventLegacy(SessionUploadE, EventFields{
+		EventID:        uuid.New(),
```
Comment (zmb3) [inline lib/events/auditlog.go]: Do you know under what circumstances this is executed? I didn't think UploadSessionRecording was ever used anymore.

File: lib/events/auditlog.go [hunk 2]
```diff
@@ -348,6 +349,7 @@ func (l *AuditLog) UploadSessionRecording(r SessionRecording) error {
 	}
 	l.log.WithFields(log.Fields{"duration": time.Since(start), "session-id": r.SessionID}).Debugf("Session upload completed.")
 	return l.EmitAuditEventLegacy(SessionUploadE, EventFields{
+		EventID:        uuid.New().String(),
```
Comment (zmb3) [inline lib/events/auditlog.go:352]: I'm 95% certain this code is never called and I will be deleting it soon.

Are you sure this was part of the bug or did you just do a global search and see it missing here too?

Comment (xacrimon) [reply lib/events/auditlog.go:352]: I did check multiple places for `session.start` that I know are used and part of the bug but I'm not sure if/how to trigger the legacy upload if it's used. I saw your comment about `session.upload` missing it and did a global search on that.

Comment (zmb3) [reply lib/events/auditlog.go:352]: Ok cool, I have no concerns with making this change here - was just wondering if you actually saw it being called.

File: lib/events/auditlog.go [hunk 3]
```diff
@@ -31,6 +31,7 @@ import (
 	"sync"
 	"time"
 
+	"github.com/google/uuid"
```
Comment (quinqu) [inline lib/events/auditlog.go:34]: I noticed we also used `github.com/pborman/uuid` to generate UUIDs (https://github.com/gravitational/teleport/blob/master/lib/auth/grpcserver.go#L235). Is there a reason we use different generators or can we just get rid of one and use the other across the project?

Comment (xacrimon) [reply lib/events/auditlog.go:34]: I'm putting up #9793 to switch to google's package across the codebase.

**Review by quinqu — APPROVED** [review]

**Review by zmb3 — APPROVED** [review]

**Pull request merged**